src/main/java/duke/ui/ExitWindow.java

```diff
@@ -0,0 +1,46 @@
+package duke.ui;
+
+import duke.MainWindow;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class ExitWindow extends HBox {
+
+    @FXML
+    private Button yesButton;
+    @FXML
+    private Button noButton;
+    @FXML
+    private Label exitMessage;
+
+    private static final String EXIT_MESSAGE = "Are you sure to exit?";
+
+    public ExitWindow() {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(MainWindow.class.getResource("/view/ExitWindow.fxml"));
```

Comment:
SLAP violation. You do object creation while the rest of the code only does method call